src/zip/zipmacro.rs

```diff
@@ -8,41 +6,30 @@
 /// This example:
 ///
 /// ```rust,ignore
-/// azip!(mut a, b, c in { *a = b + c })
+/// azip!((a in &mut a, &b in &b, &c in &c) *a = b + c);
 /// ```
 ///
 /// Is equivalent to:
 ///
 /// ```rust,ignore
-/// Zip::from(&mut a).and(&b).and(&c).apply(|a, &b, &c| {
-///     *a = b + c;
-/// });
-///
+/// Zip::from(&mut a).and(&b).and(&c).apply(|a, &b, &c| *a = b + c);
```

Comment:
I'd think that the long form with the linebreaks in the closure is easier to read as an example here — either way, let's pick the version that makes for better doc reading

Reply:
Good point, I agree. I've changed it back (except for the semicolon).